SSA/Projects/InstCombine/ForLean.lean

```diff
@@ -205,6 +216,11 @@ theorem udiv_one_eq_zero (a : BitVec w) (h : a > 1)
       assumption
     rw [Nat.div_eq_zero_iff] <;> omega
 
+lemma add_eq_xor (a b : BitVec 1) : a.add b = a.xor b := by
```

Comment:
```suggestion
@[simp]
lemma add_eq_xor (a b : BitVec 1) : a.add b = a.xor b := by
```

Reflecting on this, we can probably make this a simple lemma. xor on individual bits seems more canonical than adding over individual bits.